src/freenas/etc/crontab.template

```diff
@@ -29,6 +29,10 @@ PATH=/etc:/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/usr/local/sbin
 
 0	*	*	*	*	root	/usr/local/bin/python /usr/local/bin/mfistatus.py > /dev/null 2>&1
 
+# Run freenas-verify and save results to file
+# This helps speed up freenas-debug
+# Redmine 55752
```

Comment:
If we are doing this we should probably use /tmp to avoiding writing to boot media.

Reply:
Moved to /tmp